### ISSUE TITLE
Exclude unnecessary txs from public API

### ISSUE THREAD

**Issue by HolyGrease** [karma-coin/karmachain#116]
We can set filter for available transaction to avoid adding txs from substrate pallets that we use

**Comment by avive** [comment]
Good idea as these can be viewed in the explorer - correct?

**Comment by HolyGrease** [comment]
> Good idea as these can be viewed in the explorer - correct?
Yes, anyone can call throw polkadot ui, so I think we need to restrict this

**Comment by avive** [comment]
yes, let's restrict for now but much more important is the move to phone number hashes on chain.
